rpmbuild/SPECS/wakame-vdc.spec

```diff
@@ -445,6 +445,7 @@ mkdir -p ${RPM_BUILD_ROOT}/var/lib/%{oname}/snap
 # mussel
 mkdir -p ${RPM_BUILD_ROOT}/usr/bin
 ln -s %{prefix}/%{oname}/client/mussel/bin/mussel ${RPM_BUILD_ROOT}/usr/bin/mussel
+rsync -aHA `pwd`/client/mussel/musselrc ${RPM_BUILD_ROOT}/etc/wakame-vdc/musselrc
 
```

Comment:
Since it's in this pull request's diff, this pull request is the correct place to discuss it.